sigs/sig-marketing/README.md

```diff
@@ -0,0 +1,29 @@
+# Argo Sig Marketing
+
+Argo Sig Marketing coordinates efforts between interested parties to promote, advocate, and grow the Argo Project and community. This includes social media, subreddits, events, branding, and all other marketing activities.
+
+## Meetings
+Sig marketing meetings are held twice monthly. Meetings can be found in the [Argo Project public calendar](https://calendar.google.com/calendar/embed?src=argoproj%40gmail.com&ctz=America%2FDenver).
+
+## Membership Requirements 
+- Members are nominated and approved by Argo Proj maintainers. 
+- Must represent the open-source community’s best interest with a vendor-neutral practice.
+- Must follow the CNCF rules and guidances closely.
+- Must have shown a proven record of enabling the open-source community by marketing.
+- Technical marketing is preferred.
+
+## Membership Roles
+| Role | Responsibilities | 
+|-------------------|------------------------------------------------------|
```

Comment:
* Secretary/Coordinator - Coordinates work in the SIG. Leads and runs the SIG Marketing meetings. Reports back on SIG Marketing completed and upcoming work to maintainers.